src/main.js

```diff
@@ -55,6 +59,25 @@ const filterBeforeCreate = (toast, toasts) => {
 
 const app = createApp(App).use(store).use(router);
 
+if (
+  ["staging", "production"].includes(process.env.NODE_ENV) &&
```

Comment:
In our frontend repo, where is the NODE_ENV set?
I checked in github secrets its not there

Will also need to pass these variables in the dockerfile and then in the deployment workflows.yml as well right?

Reply:
I was under the impression that it gets sets automatically based on `npm run` or `npm build` command. Let me check.

Reply:
https://cli.vuejs.org/guide/mode-and-env.html#modes
Vue CLI sets it for us! 
That's sorted then